plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/DeltaLakeMetastore.java

```diff
@@ -45,7 +45,7 @@
 
     void createTable(ConnectorSession session, Table table, PrincipalPrivileges principalPrivileges);
 
-    void dropTable(ConnectorSession session, String databaseName, String tableName, boolean externalTable);
+    void dropTable(ConnectorSession session, String databaseName, String tableName, boolean managedTable);
```

Comment:
nit: How about renaming to `deleteData`? We need to guess the behavior from `managedTable`, but we can easily understand the behavior form `deleteData`.